app/services/forest_liana/line_stat_getter.rb

```diff
@@ -23,7 +23,7 @@ def get_format
     end
 
     def perform
-      value = get_resource().eager_load(@includes)
+      value = get_resource().eager_load(@join)
```

Comment:
It's joins if I'm right. https://scoutapm.com/blog/activerecord-includes-vs-joins-vs-preload-vs-eager_load-when-and-where
```suggestion
      value = get_resource().eager_load(@joins)
```